metrics/default_metrics/tarantool/init.lua

```diff
@@ -11,6 +11,7 @@ local default_metrics = {
     require('metrics.default_metrics.tarantool.memory'),
     require('metrics.default_metrics.tarantool.spaces'),
     require('metrics.default_metrics.tarantool.fibers'),
+    require('metrics.tarantool.vinyl')
```

Comment:
Why it doesn't stored the same as default box metrics? Is it about restructuring the repo?

Reply:
Yes